fluentd/Gemfile

```diff
@@ -5,6 +5,7 @@ gem 'elasticsearch'
 gem 'fluent-plugin-kubernetes_metadata_filter'
 gem 'fluent-plugin-concat'
 gem 'fluent-plugin-elasticsearch'
+gem 'fluent-plugin-kafka', ENV['FLUENTD_PLUGIN_KAFKA_VERSION']
```

Comment:
I might recommend the opposite change here and set the version in the Gemfile and not Dockerfile.  I don't see the advantage of it existing in the Dockerfile at all for this or fluentd unless there is some dependency there.